.semaphore/semaphore.yml

```diff
@@ -0,0 +1,87 @@
+# Use the latest stable version of Semaphore 2.0 YML syntax:
+  version: v1.0
```

Comment:
It would make life easier if we backported this to master I think, so we could make sure Semaphore 2 works fine with all our branches.